ushahidi/platform-client
# Data-view: Displaying colours on post-cards in list

This pull request makes the following changes:
- Reads the survey-colour from the correct variable

Testing checklist:
- Make sure there is a post from a survey configured with a colour 
- Go to data-view
- [ ] Look at the list to the left, the posts connected to the survey should have a colour-band on the left of the card.

Fixes ushahidi/platform#4303 (the client-side of the issue) .

Ping @ushahidi/platform
